src/Stack.mo

```diff
@@ -70,9 +97,9 @@ module {
     todo()
   };
 
-  public func mapResult<T, R, E>(stack : Stack<T>, f : T -> Result.Result<R, E>) : Result.Result<Stack<R>, E> {
-    todo()
-  };
+  // public func mapResult<T, R, E>(stack : Stack<T>, f : T -> Result.Result<R, E>) : Result.Result<Stack<R>, E> {
+  //   todo()
+  // };
 
```

Comment:
We could remove this function for now (perhaps in favor of `Iter.mapResult()` for more generality):

```suggestion
```